charts/kube-master/templates/controller-manager.yaml

```diff
@@ -90,6 +90,8 @@ spec:
             - --configure-cloud-routes=true
             - --controllers=*,bootstrapsigner,tokencleaner
             - --kubeconfig=/etc/kubernetes/config/kubeconfig
+            - --authentication-kubeconfig=/etc/kubernetes/config/kubeconfig
```

Comment:
I think this flags should be conditional, no? Otherwise the deployment will break for older clusters

Reply:
yes, thanks. I think, I'm closing this PR and rolling it into the v1.xx PR.